src/services/authService.ts

```diff
@@ -0,0 +1,32 @@
+import { errorProcessingLogin } from "functions/errorProcessing/errorProcessingLogin";
+import {
+  FirebaseClient,
+  IFirebaseClient,
+} from "./servicesClient/firebaseClient";
+import { errorProcessingRegistration } from "functions/errorProcessing/errorProcessingRegistration";
+
+class AuthService {
+  httpClient: IFirebaseClient;
+  constructor(httpClient: IFirebaseClient) {
+    this.httpClient = httpClient;
```

Comment:
Более корректное название

```suggestion
  constructor(httpClient: IFirebaseClient) {
    this.firebaseClient = firebaseClient;
```

Reply:
В итоге тебе даже не пригодится этот firbaseClient